recipe/meta.yaml

```
@@ -50,6 +55,12 @@ requirements:
     - googledrivedownloader
     - jinja2
     - pyparsing
```

Looks like we are running into an edge case with older Python versions (namely 3.8)

Added a workaround below

Asking upstream to see if we can come up with a better solution: https://github.com/conda-forge/python-feedstock/pull/668#discussion_r1523986176

```suggestion
    - pyparsing
    # Workaround to add `libxcrypt` for old Pythons
    # xref: https://github.com/conda-forge/python-feedstock/pull/668#discussion_r1523986176
    - libxcrypt  # [linux and py<39]
```